### PR TITLE
Update export to be at first level

### PR DESCRIPTION
**Pull request by thibmeu**
Emports were grouped based on internal file representation. This makes them inconvenient to import. For instance, to import `PrivateToken`, one would need to
```typescript
import { httpAuthScheme } from "@cloudflare/privacypass-ts";

httpAuthScheme.PrivateToken
```
This also makes interaction between methods hard to follow. To create such a token, you need to
```typescript
import { issuance } from "@cloudflare/privacypass-ts";

issuance.createPrivateToken(...)
```

This commit moves methods closer to their respective types, and exports types at the first level.
One can now
```typescript
import { PrivateToken } from "@cloudflare/privacypass-ts";

PrivateToken.fromInfos(...)
```
This commit also renames `TokenType` to `TOKENS_TYPES.BLIND_RSA`.

### CODE REVIEW

**Review by lukevalenta — APPROVED** [review]

File: examples/pubVerifiable.example.ts [hunk 1]
```diff
@@ -37,7 +37,12 @@ export async function publicVerifiableTokens(): Promise<void> {
     //     |<----- Request ------+                   |           |
     const redemptionContext = crypto.getRandomValues(new Uint8Array(32));
     const originInfo = ['origin.example.com', 'origin2.example.com'];
-    const tokChl = await createPrivateToken(issuer, redemptionContext, originInfo);
+    const tokChl = await PrivateToken.fromInfos(
```
Comment (armfazh) [inline examples/pubVerifiable.example.ts]: `PrivateToken.create` is a more natural name.

**Review by armfazh — APPROVED** [review]
just a comment about `create` method

File: examples/pubVerifiable.example.ts (hunk 1 above)
Comment (thibmeu) [reply examples/pubVerifiable.example.ts]: updated
